src/TodoList.js

```diff
@@ -1,5 +1,42 @@
 class TodoList {
+  constructor() {
+    this.toDoArray = [];
+  }
 
+  create(text) {
+    const todoItem = {
+      id: this.toDoArray.length + 1,
+      text: text,
+      status: "incomplete",
+    };
+
+    this.toDoArray.push(todoItem);
+
+    return todoItem;
+  }
+
+  getAll() {
+    return this.toDoArray;
+  }
+
+  setCompleteByID(ID) {
+    this.toDoArray.forEach((toDoItem) => {
```

Comment:
forEach() is new for me. 
Its seems to be a very simple solution, instead of using for loop.